pkg/adapters/postgres.go

```diff
@@ -29,10 +29,15 @@ type PostgreSQLConfig struct {
 	ServiceName   string `mapstructure:"service_name"`
 }
 
+type TuningSettings struct {
+	MemoryThreshold float64 `mapstructure:"memory_threshold" validate:"required"`
```

Comment:
This should be solved through the documentation/template, just that I normally associated the word "percentage" and the type `float` to indicate that it should be a number between `(0, 1)`, i.e. 90% is `.90`. We should make it clear that `50%` is `50`.

Reply:
Should we allow setting to 0, as in disable memory gaurdrIls? The validation is then if it's `0 < x < 1` don't allow it